website/content/en/highlights/2021-08-24-vector-aggregator.md

```diff
@@ -0,0 +1,57 @@
+---
+date: "2021-08-24"
+title: "Moving the Aggregator chart to public beta"
+description: "The Vector Aggregator Helm chart is now publicly available"
+authors: ["spencergilbert"]
+pr_numbers: [TODO]
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "announcement"
+  platforms: ["helm"]
+---
+
+We are happy to announce our *vector-aggregator* chart is now publicly available. While
```

Comment:
* s/announce/announce that
* Use backticks for vector-aggregator as it's a "code term"